### DISCUSSION TITLE
Previous Instances

### DISCUSSION THREAD

**Discussion by Direwolf343** [vrcx-team/VRCX#342] [Q&A] [answered]
Did they take this away? or is it glitched? My friend recently told me that he met me on VRChat before, so I wanted to check his previous instances to see if this was true.  Or is there another way to check the amount of times they joined you and when?

**Comment by SalbugVR** [comment, marked as answer]
Click on Join Count to view previous instances.
![image](https://user-images.githubusercontent.com/24326270/168430176-4b0a1c42-df97-4d3b-9965-73c8af02ee04.png)

**Reply by Direwolf343** [reply]
> Click on Join Count to view previous instances. ![image](https://user-images.githubusercontent.com/24326270/168430176-4b0a1c42-df97-4d3b-9965-73c8af02ee04.png)

I tried but all I see is a - 
![image](https://user-images.githubusercontent.com/105548296/168430279-0545e244-ad89-4872-8be6-de7602376ad1.png)

**Reply by SalbugVR** [reply]
Hmmm, that usually is only like that for me temporarily when I refresh my list in the friends list page but shows back up after several minutes.

**Reply by Natsumi-sama** [reply]
This information is pulled from VRCX's database so if VRCX wasn't open while in that instance it wont have this data.

**Reply by Direwolf343** [reply]
> This information is pulled from VRCX's database so if VRCX wasn't open while in that instance it wont have this data.
Ohhh thank you so much!~
